### PR TITLE
[FIX] l10n_ar_withholding: repeated taxes in chart of accounts

### PR DESCRIPTION
**Pull request by mem-adhoc**
Adhoc-task: 40503

Description of the issue/feature this PR addresses:
1- We removed the following taxes to avoid duplication:

"ex_tax_retencion_suss_sufrida", "WTH SUSS I", which is already located here(https://github.com/odoo/odoo/pull/168945/files#diff-4bca1b8ca406e3c79fa6697dcb06168a41e3a1a031fcfc467322ae0d78483523R2)

"ex_tax_retencion_ganancias_sufrida", "WTH Earnings I", which is already located here (https://github.com/odoo/odoo/pull/168945/files#diff-4bca1b8ca406e3c79fa6697dcb06168a41e3a1a031fcfc467322ae0d78483523R7) and here (https://github.com/odoo/odoo/pull/168945/files#diff-4bca1b8ca406e3c79fa6697dcb06168a41e3a1a031fcfc467322ae0d78483523R12)

2- We moved the taxes from files to match the same structure as l10n_ar

Current behavior before PR:
After downloading l10n_ar_withholding and checking the taxes, we realized there were some duplicates.
[[FIX]l10n_ar_withholding: repeated taxes in chart of accounts.webm](https://github.com/odoo/odoo/assets/109111493/56776d61-86f1-4632-98a4-c9e192cc5f3a)

Desired behavior after PR is merged:
The taxes are no longer duplicated.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by svfu-odoo — CHANGES_REQUESTED** [review]
Hello,

Thank you for the PR.
The taxes shown in the video do indeed seem to be duplicates.
But currently it is difficult to see from the diff what exactly you want to change.

(1)
Please update the commit title so that it does not fail the style check on the runbot / ci (and edit the PR title to match).
It should just be a missing space after `[FIX]`.
Also please describe the problem / change not only in the PR message but also directly in the commit message.

(2)
Please update the code to keep the diff minimal.
According to your PR message the PR is only about removing duplicates.
But in the commit you touch every line of the 2 tax template files.
I.e. you also move a lot of taxes from the `ar_ri` to the `ar_ex` template.
I don't see any justification for that in your PR message.
The `ar_ri` template inherits from the `ar_ex` template.
So I think it should probably be enough to delete the duplicates from the `ar_ri` template.

(3)
It is probably fine to do it in 17.0 (stable), but the PR is going to remove xmlids.
So we are considering whether it may not be better to do it only in master.

**Comment by vbe-odoo** [comment]
Hello @mem-adhoc thanks for the PR, I've tested and it works, I don't see the duplicated taxes, and the rest of the taxes are created properly. Can you help me answer @svfu-odoo questions: (1), (2), and (3)?

For number (3) I agree that maybe it is better to apply this FIX to master, as it is not a big deal to have those taxes duplicated, and we avoid causing unpredicted issues, but let me know what you think.

Thanks!
cc' @jco-odoo

**Comment by mem-adhoc** [comment]
Hello @svfu-odoo!
Regarding the movement of taxes, All IIBB taxes can be included in the ar_ex template (In Ri, we only add VAT.).

With this PR, the structure matches what is currently in l10n_ar for perceptions:
In l10n_ar (perceptions and other taxes):

-Most taxes are in the excento template: account.tax-ar_ex.csv
(https://github.com/odoo/odoo/blob/17.0/addons/l10n_ar/data/template/account.tax-ar_ex.csv)

-In the ri template, we add VAT: account.tax-ar_ri.csv
(https://github.com/odoo/odoo/blob/17.0/addons/l10n_ar/data/template/account.tax-ar_ri.csv)

Also, I have already corrected the name of the PR and the commit!
Greetings!

**Comment by jco-odoo** [comment]
@mem-adhoc 

- We should keep the same formatting as before in the csv. 
- It would be good to know what you delete/ what you move somehow.  (maybe put deletion in one and move in the other commit or describe it well in the commit with more than one line of explanation)

**Comment by mem-adhoc** [comment]
Hi @jco-odoo!
What do you mean by the same formatting?

**Comment by jco-odoo** [comment]
@mem-adhoc I mean that you respect the same way of adding the " e.g.  in the csv as it was before, so we see what is moved. 

Please do a rebase as well.

**Comment by vbe-odoo** [comment]
@mem-adhoc can you help me with the requested changes by @jco-odoo ?
Thanks

**Comment by mem-adhoc** [comment]
@vbe-odoo @jco-odoo 
Hi guys, good morning. Sorry for the delay, the changes you requested are now ready.

File: addons/l10n_ar_withholding/data/template/account.tax.group-ar_ex.csv [hunk 1]
```diff
@@ -1,2 +0,0 @@
-"id","name","country_id","l10n_ar_tribute_afip_code","name@es"
-"tax_group_withholding_vat","VAT Withholding","base.ar","01","Retenciones"
```
Comment (jco-odoo) [inline addons/l10n_ar_withholding/data/template/account.tax.group-ar_ex.csv:2]: We probably need to delete the one for ri

**Comment by mem-adhoc** [comment]
@jco-odoo @vbe-odoo 
Hi Jos and Valen, I have a question about the tax plan!

Is it necessary to change the prefix of the taxes in the account.tax-ar_ex.csv file from ri_ to ex_?"

**Comment by vbe-odoo** [comment]
> @jco-odoo @vbe-odoo Hi Jos and Valen, I have a question about the tax plan!
> 
> Is it necessary to change the prefix of the taxes in the account.tax-ar_ex.csv file from ri_ to ex_?"



> @jco-odoo @vbe-odoo Hi Jos and Valen, I have a question about the tax plan!
> 
> Is it necessary to change the prefix of the taxes in the account.tax-ar_ex.csv file from ri_ to ex_?"

I don't have an strong opinion on this, maybe it is better to change from `ri_ to ex_` not sure if @jjscarafia has an opinion on this one?

Thanks!

**Comment by jjscarafia** [comment]
I agree @vbe-odoo , better to change from ri_ to ex_
@vbe-odoo @mem-adhoc

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**